lib/filesystem/FileStream.cpp

```diff
@@ -91,37 +91,34 @@ static voidpf ZCALLBACK fopen64_file_func(voidpf opaque, const void* filename, i
 			if (mode & ZLIB_FILEFUNC_MODE_CREATE)
 				mode_fopen = "wb";
 
-	if ((filename != NULL) && (mode_fopen != NULL))
+	if ((filename != nullptr) && (mode_fopen != nullptr))
 		file = FOPEN_FUNC((const char*)filename, mode_fopen);
 	return file;
 }
 
 
 static uLong ZCALLBACK fread_file_func(voidpf opaque, voidpf stream, void* buf, uLong size)
 {
-	uLong ret;
-	ret = (uLong)fread(buf, 1, (size_t)size, (FILE *)stream);
+	auto ret = (uLong)fread(buf, 1, (size_t)size, (FILE *)stream);
 	return ret;
 }
 
 static uLong ZCALLBACK fwrite_file_func(voidpf opaque, voidpf stream, const void* buf, uLong size)
 {
-	uLong ret;
-	ret = (uLong)fwrite(buf, 1, (size_t)size, (FILE *)stream);
+	auto ret = (uLong)fwrite(buf, 1, (size_t)size, (FILE *)stream);
 	return ret;
 }
 
 static ZPOS64_T ZCALLBACK ftell64_file_func(voidpf opaque, voidpf stream)
 {
-	ZPOS64_T ret;
-	ret = FTELLO_FUNC((FILE *)stream);
+	ZPOS64_T ret = FTELLO_FUNC((FILE *)stream);
 	return ret;
 }
 
 static long ZCALLBACK fseek64_file_func(voidpf  opaque, voidpf stream, ZPOS64_T offset, int origin)
 {
 	int fseek_origin = 0;
-	long ret;
+	long ret = 0;
```

Comment:
Can you remove it entirely? It is basically unused anyway.
```
Line #137: ret = -1;   -> return -1;
Line #139: return ret; -> return 0;
```

Reply:
done